src/pages/index.js

```diff
@@ -21,73 +21,229 @@ import {
   profilePicture,
   openModal,
   closePopup,
+  profileAvatarEditButton,
   options,
 } from "../utils/constants.js";
+import Api from "../components/Api.js";
+import PopupWithConfirmation from "../components/PopUpWithConfirmation.js";
 
 // Data
 const initialCards = [
```

Comment:
From the 8th sprint all constants (including `initialCards` and `settings`) should be in file `constants.js` in folder `utils` to clean up the main file.